README.md

```diff
@@ -12,23 +13,33 @@ If you have a question about the Istio Manager or have a problem using it, pleas
 
 ## Build instructions ##
 
-We are using [Bazel 0.4.2](https://bazel.io) on Linux to build Istio Manager:
+We are using [Bazel 0.4.3](https://bazel.io) to build Istio Manager:
 
-    bazel build :all
+    bazel build //...
 
-You can also use regular `go test` and `go build` but keep in mind
-that we are building against a specific version of [Kubernetes `client-go`](https://github.com/kubernetes/client-go).
-Place the repository clone into your `$GOROOT`:
+_Note_: Due to issues with case-insensitive file systems, macOS is not
+supported at the moment.
 
-    mkdir -p $GOROOT/src/istio.io
-    ln -s <istio.io/manager repo> $GOROOT/src/istio.io
+You can also use regular `go test` and `go build` if you place your repository
+into `$GOPATH`:
+
+    mkdir -p $GOPATH/src/istio.io
+    ln -s <istio.io/manager repo> $GOPATH/src/istio.io
     go test ./... -v
 
+You would need to `go get` each dependency at the right SHA (see `WORKSPACE` file).
+
 ## Test environment ##
 
-We are using [minikube](https://github.com/kubernetes/minikube) to develop for the Kubernetes environment.
-To let Bazel sandbox access the cluster, please add a symbolic link to your configuration:
+Manager tests require an access to a Kubernetes cluster. Each test creates a
+temporary namespace and deletes it on completion.  Please configure your
+`kubectl` to point to a development cluster before invoking the tests.
+
+To let Bazel sandboxes access the cluster, please add a symbolic link to your
+repository pointing to to your Kubernetes configuration file:
 
     ln -s ~/.kube/config platform/kube/
-    bazel test :all
+    bazel test //...
 
+_Note_: Due to a well-known issue, the namespaces are not deleted completely
+after running the tests and permanently reside in a terminating state.
```

Comment:
Added. Apparently it got fixed in the last few weeks, but the change is not released.